integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/InfinispanStoreJdbcIT.java

```diff
@@ -32,7 +34,10 @@
 
    @Deployment
    public static Archive<?> deployment() {
-      return ShrinkWrap.create(WebArchive.class, "jdbc.war").addClass(InfinispanStoreJdbcIT.class).add(manifest(), "META-INF/MANIFEST.MF");
+      return ShrinkWrap.create(WebArchive.class, "jdbc.war")
+            .addClass(InfinispanStoreJdbcIT.class)
+            .addAsResource("jdbc-config.xml")
+            .add(manifest(), "META-INF/MANIFEST.MF");
```

Comment:
I think the org.infinispan:main module should import all optional store modules.